crawl-ref/docs/options_guide.txt

```diff
@@ -459,6 +460,14 @@ pregen_dungeon = incremental
         level entry, as was the rule before 0.23. Dungeons will not be stable
         given a seed with this option.
 
+suppress_startup_errors = false
```

Comment:
Thanks for fixing up these various missing/out of date option descriptions! In future, it'd probably be better to put these types of fixes in a separate, smaller PR, to make review easier.

Reply:
Would it help to split it now? I only put them there because one of the other changes draws attention to the missing options.

Reply:
I think it'd be nice - I suspect we'd be able to merge the options fixes much faster than the rest of this commit. Don't do it if it's a ton of work for you, though.

Reply:
I've done this now.